uw_manipulation/README.md

```diff
@@ -0,0 +1,3 @@
+# fetch_api
```

Comment:
Looks like a copy paste leftover, but it would be good to have a stub readme with a sentence about what's here